### DISCUSSION TITLE
Introducing Angular Mini Blog Series

### DISCUSSION THREAD

**Discussion by ShreyaPrasad1209** [EddieHubCommunity/support#2029] [Show and tell]
Hey folks 👋🏻 

I have started a mini blog series on [Dev.to](https://dev.to/shreyapd06/introducing-angular-mini-blog-series-part-1-4hdg) ✍🏻 
This is my first time writing on Dev.to and that too a series so I would love to hear any constructive feedbacks, areas of improvements 💭 

Also, please like the blog if you love it ❤️

**Comment by ryancraigmartin** [comment]
Hi @ShreyaPrasad1209! Mind linking to the new series so we can check it out? I couldn't find you with your github username.
